cmd/api/api.go

```diff
@@ -5,16 +5,19 @@ import (
 	"crypto/tls"
 	"fmt"
 	"net/http"
+	"net/url"
 	"os"
 	"os/signal"
+	"strings"
 	"syscall"
 	"time"
 
 	"sda-pipeline/internal/broker"
 	"sda-pipeline/internal/config"
 	"sda-pipeline/internal/database"
 
-	"github.com/gorilla/mux"
+	"github.com/gin-gonic/gin"
+	"github.com/golang-jwt/jwt"
```

Comment:
i recommend https://github.com/lestrrat-go/jwx it is better coverage and maintenance + we have it in sda-download already as we need jwk so it is better to have one library instead of multiple

Reply:
I am working on changing to that, not the nicest task, but I should be done soon :)